internal/weldr/api.go

```diff
@@ -1411,8 +1411,25 @@ func (api *API) composeHandler(writer http.ResponseWriter, request *http.Request
 			statusResponseError(writer, http.StatusBadRequest, errors)
 			return
 		}
+		targets = append(targets, t)
 	}
 
+	filename, _, err := api.distro.FilenameFromType(cr.ComposeType)
```

Comment:
`distro.FilenameFromType()` is deprecated. Simply use `imageType.Filename()` instead, and you can drop the error handling.

Reply:
Oh I get to drop the whole error case. Nice!